pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchurl, lib, stdenv, gnome3, cmake, pkg-config,
+libappindicator-gtk3, gst_all_1, pcre }:
+
+stdenv.mkDerivation rec {
+  pname = "whatsapp-for-linux";
+  version = "1.1.5";
+
+  src = fetchurl {
+    url = "https://github.com/eneshecan/whatsapp-for-linux/archive/v${version}.tar.gz";
+    sha256 = "6e034d9f7d1dca94df943276ce91227088bc986442ee095d0904f68fb7abbf44";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    gnome3.gtkmm
+    gnome3.webkitgtk
+    libappindicator-gtk3
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-libav
+    pcre
+  ];
+
+  dontStrip = true;
+  dontPatchELF = true;
+
+  installPhase = ''
+  runHook preInstall
+
+  cmake -DCMAKE_BUILD_TYPE=Release .
+  make
+  make install
+  runHook postInstall
+  '';
```

Comment:
Letting stdenv do it doesn't work?

Reply:
It works, I removed the custom installPhase